src/inc/msquic_winuser.h

```diff
@@ -250,10 +250,35 @@ QuicAddrSetToLoopback(
     )
 {
     if (Addr->si_family == QUIC_ADDRESS_FAMILY_INET) {
+
+#ifdef DUONIC_TESTING
```

Comment:
These are our public API headers that we ship. I'd prefer a solution that doesn't modify them for DuoNic

Reply:
Maybe define test-only wrappers (msquichelper.h?) that does what you want, but only is called by test code.

Reply:
I've updated the test code to override LOCALHOST_FOR_AF, and to rewrite the QuicAddrs that are created with a boolean param (AKA the loopback ones), to remove this stuff from the msquic API header.